crates/bindings-sys/src/lib.rs

```diff
@@ -50,7 +49,7 @@ pub mod raw {
         /// - the slice `(name, name_len)` is not valid UTF-8
         /// - `name + name_len` overflows a 64-bit address.
         /// - writing to `out` overflows a 32-bit integer
-        pub fn _get_table_id(name: *const u8, name_len: usize, out: *mut TableId) -> u16;
```

Comment:
Why are we changing all of the underscores here? This is not mentioned in the proposal and I think is best done in a different proposal and/or PR.

Reply:
Since this is changing ABI, let's keep this diff as small as possible. We can agree on different naming in the ABI stabilization proposal.